pytition/petition/templates/petition/base.html

```diff
@@ -30,14 +38,16 @@
         max-width: 35px;
     }
     </style>
+
+    <!-- Extra js in head -->
+    {% block extrajshead %}
+    {% endblock %}
 </head>
 <body>
-<div id="modalContainer">
+    <div id="modalContainer">
 
-</div>
-{% block container_type %}
```

Comment:
Is there a reason to pull the `<nav>` bar outside the container{-fluid}?
I liked the rendering of the `<nav>` bar which is not full-width and with a bit of space under it (that was probably thanks to the `<p></p>` which was under it)

Reply:
The non full width bar is due to the padding in container div. I prefer to avoid this spacing, but if you don't I can put it back in container{-fluid} :).